rec_to_nwb/processing/builder/raw_to_nwb_builder.py

```diff
@@ -124,13 +124,14 @@ def __is_rec_config_valid(self):
             if self.trodes_rec_export_args[i] == '-reconfig':
                 xml_file_path = self.trodes_rec_export_args[i + 1]
 
-        validation_registrator = ValidationRegistrator()
-        validation_registrator.register(XmlFilesValidator(xml_file_path))
-        validation_registrator.validate()
-
-        xsd_file_path = str(path) + '/../../../rec_to_nwb/data/reconfig_header.xsd'
-        xsd_schema = xmlschema.XMLSchema(xsd_file_path)
-        xmlschema.validate(xml_file_path, xsd_schema)
+        if xml_file_path:
+            validation_registrator = ValidationRegistrator()
+            validation_registrator.register(XmlFilesValidator(xml_file_path))
+            validation_registrator.validate()
+
+            xsd_file_path = str(path) + '/../../../rec_to_nwb/data/reconfig_header.xsd'
```

Comment:
The user specifies not xsd, but xml file, so if the header is not specified in the _DEFAULT_TRODES_REC_EXPORT_ARGS then it won't work at all.